tasks.py

```diff
@@ -47,15 +69,23 @@ def install_minimum(c):
     for line in lines:
         if started:
             if line == ']':
-                break
+                started = False
+                continue
 
             line = line.strip()
-            line = re.sub(r',?<=?[\d.]*,?', '', line)
-            line = re.sub(r'>=?', '==', line)
-            line = re.sub(r"""['",]""", '', line)
-            versions.append(line)
-
-        elif line.startswith('install_requires = ['):
+            if _validate_python_version(line):
+                requirement = re.match(r'[^>]*', line).group(0)
+                requirement = re.sub(r"""['",]""", '', requirement)
+                version = re.search(r'>=?(\d\.?)+', line).group(0)
```

Comment:
I came out with a better regex for this in this branch: 
https://github.com/sdv-dev/SDV/blob/c3ccfb071d94f22dc322514f2394ab398fff7d46/tasks.py#L54 . Maybe we can use this here too